routing/missioncontrol.go

```diff
@@ -224,11 +231,12 @@ func NewMissionControl(db kvdb.Backend, self route.Vertex,
 	}
 
 	mc := &MissionControl{
-		state:     newMissionControlState(cfg.MinFailureRelaxInterval),
-		now:       time.Now,
-		selfNode:  self,
-		store:     store,
-		estimator: cfg.Estimator,
+		state:           newMissionControlState(cfg.MinFailureRelaxInterval),
```

Comment:
This now needs to be formatted differently because it exceeds the allowed line length.
For example:
```
	mc := &MissionControl{
		state: newMissionControlState(
			cfg.MinFailureRelaxInterval,
		),
		now:             time.Now,
		selfNode:        self,
		store:           store,
		estimator:       cfg.Estimator,
		UpdateEstimator: cfg.UpdateEstimator,
	}
```